platform/ioscross/README.md

```diff
@@ -0,0 +1,52 @@
+Cross compiling iOS from Linux
+==============================
+
+This directory contains scripts for compiling a iOS version 
+of Csound from Linux.  These scripts were developed using Ubuntu.
+
+**[Please ensure you have read and understood the Xcode license
+terms first before using it.](https://www.apple.com/legal/sla/docs/xcode.pdf)**
+
+## Instructions
+
+1. Install docker using apt:
+
+```bash
+sudo apt install docker.io
+```
+
+2. Add user to docker group:
+
+```bash
+sudo usermod -a -G docker $USER
+```
+
+3. Download XCode with the iOS SDK (you must be logged into an Apple ID to download Xcode).
```

Comment:
Are we using the SDKs or Xcode for this cross compilation at some point?

Reply:
Yes, Xcode and the SDKs are used but the end user is responsible getting an Apple ID and downloading Xcode.  The scripts themselves do not download Xcode to avoid re-distribution of the Apple SDK.